README.md

```diff
@@ -41,18 +41,199 @@ action:
+
   schema_handler = SchemaHandler()
-  schema_handler.infer_table_schema_from_pandas_dataframe(df)
+  table_schema = schema_handler.infer_table_schema_from_pandas_dataframe(dataframe)
+  ```
+#### 2. Upload Aito schema 
+
+To upload your Aito schema, you will need the name of your Aito instance and the read-write API key.
+
+##### Upload Aito table schema
+  ```python
+  from aito.utils.aito_client import AitoClient
+
+  aito_table_name = "your-table-name-in-aito-schema"
+
+  # Aito table schema example
+  table_schema = {
+      "type" : "table",
+      "columns": {
+          "Churn": {
+              "nullable": True,
+              "type": "Boolean"
+          },
+          "CustomerID": {
+              "nullable": False,
+              "type": "String"
+              }
+          }
+      }
+
+  aito_client = AitoClient(instance_name="your-aito-instance-name", api_key="your-rw-api-key")
+  aito_client.put_table_schema(table_name=aito_table_name, table_schema=table_schema)
+
+  # Check your table schema in Aito
+  aito_client.get_table_schema(table_name=aito_table_name)
+  ```
+
+##### Upload full Aito schema
+  ```python
+  from aito.utils.aito_client import AitoClient
+
+  # Aito DB schema example
+  database_schema = {
+      "schema": {
+        "Churntable": {
+          "columns": {
+            "Churn": {
+              "nullable": true,
+              "type": "Boolean"
+            },
+            "CustomerID": {
+              "nullable": false,
+              "type": "String",
+              "link": "Customertable.CustomerID"
+            }
+          }
+        },
+        "Customertable": {
+          "columns": {
+            "CustomerID": {
+              "type": "String"
+            },
+            "name": {
+              "type": "String"
+            }
+          }
+        }
+      }
+    }
+
+  aito_client = AitoClient(instance_name="your-aito-instance-name", api_key="your-rw-api-key")
+  aito_client.put_database_schema(database_schema=database_schema)
+
+  # Check your DB schema in Aito
+  aito_client.get_database_schema()
+  ```
+
+#### 3. Upload data
+
+To get your data into Aito, you will first have to transform it into the JSON format. You can use the library to either upload your data by reading it into memory or by file upload (yielding is not currently supported). For file upload, the file has to be in gzip compressed newline delimited JSON format (file should end with .ndjson.gz). You will also need the name of your Aito instance and the read-write API key.
+
+You can download the example file from [here](https://raw.githubusercontent.com/AitoDotAI/kickstart/master/reddit_sample.csv).
+
+##### Upload data in memory
+  ```python
+  import pandas as pd
+  from aito.utils.aito_client import AitoClient
+
+  # Define the table you want to upload data into
+  aito_table_name = "your-table-name-in-aito-schema"
+
+  # Load a CSV into a pandas Dataframe
+  dataframe = pd.read_csv("reddit_sample.csv", delimiter=";")
```

Comment:
```suggestion
  dataframe = pd.read_csv("reddit_sample.csv")
```

Reply:
";" Delimiter messes up a normal csv and should only be used when it's formatted differently

Reply:
I'll change it to "," in the next commit. I'll make all the changes into one commit. I think it is still good to have the delimiter specified there.